README.md

```diff
@@ -1,6 +1,44 @@
 # Enarx
-A client for deploying WebAssembly applications into Enarx Keeps.
 
-This is currently a placeholder repository.
+## Introduction
+Enarx is an application deployment system enabling applications to run within Trusted Execution Environments (TEEs) without rewriting for particular platforms or SDKs. It handles attestation and delivery into a run-time “Keep” based on WebAssembly, offering developers a wide range of language choices for implementation. Enarx is CPU-architecture independent, enabling the same application code to be deployed across multiple targets, abstracting issues such as cross-compilation and differing attestation mechanisms between hardware vendors. Work is currently underway on AMD SEV and Intel SGX.
+
+We've known for a long time that we need encryption for data at rest and in transit: Enarx helps you do encryption for data in use.
+
+[Read more](https://github.com/enarx/enarx/wiki/Enarx-Introduction)
+
+## Project aim
+Create a way to create and run "private, fungible, serverless" applications using Trusted Execution Environments (TEEs). In other words, to provide a platform abstraction for TEEs.
+
+They should:
+
+ * be confidential from the host and other workloads;
+ * be able to run on
+   * multiple silicon architectures; 
+   * multiple server types;
+   * public cloud or private cloud;
+ * employ serverless deployment and execution mechanisms.
```

Comment:
Rather than the buzzword I'd describe the properties this implies in your understanding.

Reply:
This is a good point @steveeJ, however i think that will require to put thought into it than what this issue and PR intended to do. I'll resolve for now but open an issue to track this.